src/Simulation/DepletionVoltage.jl

```diff
@@ -25,13 +25,12 @@ Returns the potential (in V) needed at the [`Contact`](@ref) with id `contact_id
 to fully deplete the detector in a given [`Simulation`](@ref). For this, all other
 contact potentials are set to `0` and the potential at the specified contact is
 increased or decreased according to the `potential_range`. The depletion voltage
-is set to the potential for which a previously undepleted detector becomes depleted,
-or for which a previously depleted detector becomes undepleted.
```

Comment:
I would like to have this method work like explained in the previous docstring.
The potential range should be allowed to be increasing or decreasing, meaning that we can either start depleting an undepleted detector or undepleting a depleted detector.